lib/screens/gh_repo.dart

```diff
@@ -333,7 +333,12 @@ class GhRepoScreen extends StatelessWidget {
                       },
                     ),
                     url: '/github/$owner/$name/contributors',
-                  )
+                  ),
+                  TableViewItem(
+                    leftIconData: Octicons.book,
+                    text: Text("Releases"),
+                    url: '/github/$owner/$name/releases',
+                  ),
```

Comment:
1. We could have releases count here.
2. The releases entry in the action sheet (top-right menu) could be removed

Reply:
I noticed there's projects also. I'll add a screen for that as well.